ui/app/AppLayouts/ActivityCenter/ActivityCenterLayout.qml

```diff
@@ -36,6 +36,8 @@ StatusSectionLayout {
     required property NotificationsStore notificationsStore
     required property MessagingStores.MessagingRootStore messagingRootStore
 
+    signal navToMsgDetailsRequested(bool navigate)
```

Comment:
Please add not it's temporary solution. And maybe some explanation somewhere why we need it 🙏